src/dune_file.ml

```diff
@@ -1894,20 +1893,34 @@ module Copy_files = struct
 end
 
 module Documentation = struct
+  module Doc_Syntax = struct
+    type t = Reason | OCaml | All
```

Comment:
Other places use something like `All | Only of syntax list` which is more flexible - with this type you can not only enable a subset, it's all or only one. But this is not a problem since there are only two languages here.

Reply:
I realized that `All` doesn't make sense anyways. `odoc` plans to provide both formats within the same html file, so the syntax option is only for the "preferred default syntax" later on